students/yechan/westagram/users/views.py

```diff
@@ -36,13 +38,21 @@ class SingInView(View):
     def post(self, request):
         try:
             data     = json.loads(request.body)
-            email    = data['email']
-            password = data['password']
+            user     = User.objects.get(email=data['email'])
+
+            if not bcrypt.checkpw(data['password'].encode('utf-8'), user.password.encode('utf-8')):
+                return JsonResponse({'message':'INVALID_PASSWORD'},status=401)
+
+            access_token = jwt.encode({'id': user.id}, settings.SECRET_KEY, algorithm = settings.ALGORITHM)
 
-            if User.objects.filter(email=email,password=password).exists():
-                return JsonResponse({'message':'SUCCESS'},status=201)
-            else:
-                 return JsonResponse({'message':'INVALID_USER'},status=401)
+            return JsonResponse({
+                    'message'      : 'SUCCESS',
+                    'access_token' : access_token,
+                },status=200)
```

Comment:
```suggestion
            return JsonResponse({'access_token' : access_token},status=200)
```
성공의 결과로 토큰을 주기 때문에 메시지는 생략해도 될 것 같습니다.